deploy/olm-catalog/ocs-operator/manifests/ocs-operator.v4.5.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1586 @@
+                  "namespace": "openshift-storage"
+              },
+              "spec": {
+                  "manageNodes": false,
+                  "monPVCTemplate": {
+                      "spec": {
+                          "accessModes": [
+                              "ReadWriteOnce"
+                          ],
+                          "resources": {
+                              "requests": {
+                                  "storage": "10Gi"
+                              }
+                          },
+                          "storageClassName": "gp2"
+                      }
+                  },
+                  "storageDeviceSets": [
+                      {
+                          "count": 3,
+                          "dataPVCTemplate": {
+                              "spec": {
+                                  "accessModes": [
+                                      "ReadWriteOnce"
+                                  ],
+                                  "resources": {
+                                      "requests": {
+                                          "storage": "1Ti"
+                                      }
+                                  },
+                                  "storageClassName": "gp2",
+                                  "volumeMode": "Block"
+                              }
+                          },
+                          "name": "example-deviceset",
+                          "placement": {},
+                          "portable": true,
+                          "resources": {}
+                      }
+                  ]
+              }
+          }
+      ]
+    capabilities: Full Lifecycle
+    categories: Storage
+    containerImage: quay.io/ocs-dev/ocs-operator:4.5.0
+    createdAt: "2020-05-04 13:31:04"
+    description:
+      Red Hat OpenShift Container Storage provides hyperconverged storage
+      for applications within an OpenShift cluster.
+    external.cluster.ocs.openshift.io/supported: "true"
+    features.ocs.openshift.io/enabled: '["external"]'
+    olm.skipRange: ">=0.0.1 <4.5.0"
+    operatorframework.io/cluster-monitoring: "true"
+    operatorframework.io/suggested-namespace: openshift-storage
+    operators.operatorframework.io/internal-objects:
+      '["cephclusters.ceph.rook.io",
+      "cephblockpools.ceph.rook.io", "cephobjectstores.ceph.rook.io", "cephobjectstoreusers.ceph.rook.io",
+      "cephnfses.ceph.rook.io","cephclients.ceph.rook.io", "noobaas.noobaa.io", "objectbuckets.objectbucket.io","objectbucketclaims.objectbucket.io","ocsinitializations.ocs.openshift.io",
+      "storageclusterinitializations.ocs.openshift.io"]'
+    repository: https://github.com/openshift/ocs-operator
+    support: Red Hat
+  name: ocs-operator.v4.5.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+          OCS Initialization represents the initial data to be created when
```

Comment:
I think the indentation here is wrong. I get an error when building the container image:
```
 [could not decode contents of file /registr
y/ocs-catalog/ocs-operator/bundle.Dockerfile into package: error unmarshaling JSON: while decoding JSON: json: cannot unmarshal string into Go valu
e of type registry.PackageManifest, could not decode contents of file /registry/ocs-catalog/ocs-operator/manifests/ocs-operator.v4.5.0.clusterservi
ceversion.yaml into package: error converting YAML to JSON: yaml: line 84: mapping values are not allowed in this context]"
```

Reply:
Yes there are a few places where indentation/keys are incorrect.

Reply:
@estroz It's not modified by hand. This is what I got after creating bundle using sdk.
It's supposed to be exact copy of what we had before. I'll take a look at why it changed.